packages/gatsby/cache-dir/fast-refresh-overlay/utils.js

```diff
@@ -7,11 +7,21 @@ export function prettifyStack(errorInformation) {
   } else {
     txt = errorInformation
   }
-  return Anser.ansiToJson(txt, {
+  const generated = Anser.ansiToJson(txt, {
     remove_empty: true,
     use_classes: true,
     json: true,
   })
+  // Sometimes the first line/entry is an "Enter", so we need to filter this out
+  const [firstLine, ...rest] = generated
+  if (
+    firstLine.content ===
+    `
+`
+  ) {
```

Comment:
Would this work on windows?

My suggestion would be to go for something along these lines.
```suggestion
  if (/\r?\n/.test(firstLine.content)) {
```

Reply:
So in my testing, when I used this regex, it didn't work anymore.

Reply:
When I used your suggestion the errors didn't correctly show up anymore in my testing.

Reply:
Ok, the correct regex is `/^\s$/`.

```
^ anchor begin of string
\s whitespace character class (space, tab, newline)
$ end of string
```